src/services/lomakkeet/perustelut/muut.js

```diff
@@ -598,13 +598,43 @@ export const getOppisopimusPerusteluLomake = isReadOnly => {
   ];
 };
 
+export const generateDifferenceComponent = ({
+  titles,
+  changeObject = {},
+  isReadOnly = false,
+  isRequired = false,
+  name = ''
+}) => {
+  return {
+    anchor: "A",
+    name: "Difference",
+    properties: {
+      initialValue: changeObject.initialValue || 0,
+      applyForValue: changeObject.applyForValue || 0,
+      titles,
+      isReadOnly,
+      isRequired,
+      name
+    }
+  }
+};
+
 export const getOpiskelijavuodetVahimmaisopiskelijavuosimaaraPerustelulomake = (
   checkboxItems,
   locale,
-  isReadOnly = false
+  isReadOnly = false,
+  changeObject,
+  differenceComponentTitles
 ) => {
   const checkboxes = getMuutostarveCheckboxes(checkboxItems, locale, isReadOnly);
   return [
+    {
+      anchor: "vahimmaisopiskelijavuodet",
+      title: "Haettava määrä",
```

Comment:
Localization should be done.

Reply:
Not in scope of this issue. Localisation should be done for all matter in muut.js. Separate, bigger issue.